source/install/install-ubuntu-1604-mattermost.rst

```diff
@@ -81,13 +81,14 @@ Assume that the IP address of this server is 10.10.10.2.
     WorkingDirectory=/opt/mattermost
     User=mattermost
     Group=mattermost
+    LimitNOFILE=49152
```

Comment:
I copied the value from the RHEL 7.1 install docs. @coreyhulen Can you confirm that this is ok, or should I change it?